js/samples/04.ai-apps/d.assistants-mathBot/infra/azure.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceBaseName": {
+            "value": "bot${{RESOURCE_SUFFIX}}"
+        },
+        "botAadAppClientId": {
+            "value": "${{BOT_ID}}"
+        },
+        "botAadAppClientSecret": {
+            "value": "${{SECRET_BOT_PASSWORD}}"
+        },
+        "webAppSKU": {
+            "value": "B1"
+        },
+        "botDisplayName": {
+            "value": "ChatBotModeration"
```

Comment:
this might be across all the PRs but rename value to `MathBot` here

Reply:
Thanks for the catch

Reply:
Looks like it was just this sample